[Publish Oct 8th] MCP Tools (#7981)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/about-mcp.md

- `generate_model_yaml`: Generates documentation YAML for existing dbt models, including column names, data types, and description placeholders.
- `generate_staging_model`: Creates staging SQL models from sources to transform raw source data into clean staging models.

### Fusion tools (Remote)

> A set of tools, leveraging the <Constant name="fusion"> engine.

Reviewer: ```suggestion
A set of tools, leveraging the <Constant name="fusion" /> engine.
```

> A set of tools, leveraging the <Constant name="fusion" /> engine.

Reviewer: ```suggestion
A set of tools that leverage the <Constant name="fusion" /> engine for advanced SQL compilation and column-level lineage analysis.
```

> - `compile_sql`: Compile a SQL statement in the context of the current project and environment.

Reviewer: ```suggestion
- `compile_sql`: Compiles a SQL statement in the context of the current project and environment.
```

- `get_column_lineage`: TODO

### Fusion tools (Local)

> A set of tools, leveraging the <Constant name="fusion"> engine with access to a locally running <Constant name="fusion"> LSP when the VS Code extension is installed:

Reviewer: ```suggestion
A set of tools, leveraging the <Constant name="fusion" /> engine with access to a locally running <Constant name="fusion"> LSP when the VS Code extension is installed:
```

> A set of tools, leveraging the <Constant name="fusion" /> engine with access to a locally running <Constant name="fusion"> LSP when the VS Code extension is installed:

Reviewer: ```suggestion
A set of tools, leveraging the <Constant name="fusion" /> engine with access to a locally running <Constant name="fusion" /> LSP when the VS Code extension is installed:
```

> A set of tools, leveraging the <Constant name="fusion" /> engine with access to a locally running <Constant name="fusion" /> LSP when the VS Code extension is installed:

Reviewer: ```suggestion
A set of tools that leverage the <Constant name="fusion" /> engine through a locally running <Constant name="fusion" /> Language Server Protocol (LSP) in VS Code or Cursor with the dbt VS Code extension.
```

> - `get_column_lineage`: <Constant name="fusion" /> exclusive! Get column lineage information across a project DAG for a specific column.

Reviewer: ```suggestion
- `get_column_lineage`: <Constant name="fusion" /> exclusive! Gets column lineage information across a project DAG for a specific column.
```
